### ISSUE TITLE
Android Gradle Plugin version 3.3.0-beta04 problem

### ISSUE THREAD

**Issue by zsavely** [johncarl81/parceler#365]
The library can't be used with Android Gradle Plugin version 3.3.0-beta04 as Jetifier refuses to transform it.

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':telemed-app:patientMegafonDebugAnnotationProcessorClasspath'.
> Failed to transform file 'parceler-1.1.11.jar' to match attributes {artifactType=processed-jar} using transform JetifyTransform
   > Failed to transform '/home/savelii/.gradle/caches/modules-2/files-2.1/org.parceler/parceler/1.1.11/98501bf35dc8296db0e31146ddd26c57b3952ef4/parceler-1.1.11.jar' using Jetifier. Reason: The given artifact contains a string literal with a package reference 'android.support.v4.widget' that cannot be safely rewritten. Libraries using reflection such as annotation processors need to be updated manually to add support for androidx.. (Run with --stacktrace for more details.)
```

**Comment by johncarl81** [comment]
See https://github.com/johncarl81/parceler/issues/362, Already fixed, just need to release 1.1.12.

**Comment by johncarl81** [comment]
... in the mean time you can use 1.1.12-SNAPSHOT which is up on maven central.

**Comment by johncarl81** [comment]
Ok, just released 1.1.12 with this fix.
